charts/fluentd/templates/_pod.tpl

```diff
@@ -47,15 +47,15 @@ containers:
       httpGet:
         path: /metrics
         port: metrics
-    {{- if .Values.livenessProbeConfig }}
-      {{- toYaml .Values.livenessProbeConfig | nindent 6 }}
+    {{- if .Values.livenessProbe }}
+      {{- toYaml .Values.livenessProbe | nindent 6 }}
```

Comment:
I think the probe values should be able to replace the current probes, not just configure the non-specified settings (timeoutSec, failureThreshold and so on).

The easiest way would be to move the `httpGet` stuff here into the values.yaml file

Reply:
I'm fairly confident that the values will be overwriten if you specify them on the values.
For example, if the template has:

```yaml
    readinessProbe:
      httpGet:
        path: /metrics
        port: metrics
```

They can be overwritten if the values.yaml as:

```yaml
    readinessProbe:
      httpGet:
        path: /metricsOverwrite
```

Reply:
Although, thinking about what @naseemkullah said it makes sense to pass them to values.yaml, because if you deactivate the prometheus-metrics the probes will stop working.

Maybe we can improve this in the future to be more "automatic"

Reply:
please check the latest commit. should be ok now